docs/circuits.ipynb

```diff
@@ -754,10 +744,20 @@
     "id": "7af5e8ea5b45"
    },
    "source": [
-    "### Related\n",
+    "# Summary\n",
+    "\n",
+    "`Circuit`s are sliceable iterables of `Moments`, which are sliceable iterables of `Operation`s, which are `Gate`s applied to `Qubit`s. Cirq provides intuitive and flexible ways to construct, add to, divide and nest `Circuits`, with generators, slicing and `CircuitOperation`s. \n",
```

Comment:
How do we feel about the Oxford comma?
`construct, add to, divide and nest`
-> `construct, add to, divide, and nest`
`generators, slicing and CircuitOperations`
-> `generators, slicing, and CircuitOperations`

Reply:
Better.